spec/models/conference_spec.rb

```diff
@@ -41,6 +41,16 @@
       end
     end
   end
+  
+  describe '#dates' do
+    subject { FactoryGirl.build_stubbed(:conference) }
+    
+    it 'has a date range' do
```

Comment:
(The Pipeline Operator Influence   `|>` is strong in this one :wink: :heart_decoration:)